source/classes.tex

```diff
@@ -2260,10 +2260,9 @@
 \pnum
 \indextext{destructor!virtual}%
 \indextext{destructor!pure virtual}%
-A prospective destructor can be declared
-\tcode{virtual}\iref{class.virtual}
-or pure
-\tcode{virtual}\iref{class.abstract}.
+A prospective destructor can be
+a virtual\iref{class.virtual}
+or pure virtual function\iref{class.abstract}.
 If the destructor of a class is virtual and
 any objects of that class or any derived class are created in the program,
 the destructor shall be defined.
```

Comment:
Side note (separate pull request, please): I think this "shall be defined" business should be / is covered by [basic.def.odr], so this can become a note.

Reply:
Will do.

Reply:
#3966